config/sync/views.view.user_content.yml

```diff
@@ -208,7 +208,7 @@ display:
           empty: true
           tokenize: false
           content:
-            value: 'I don''t have any content at the moment. Please come back soon... '
+            value: '<h3>I don''t have any content at the moment. Please come back soon...</h3>'
```

Comment:
Add the h3 or subheading class, it's still not clear enough

Reply:
it's got the styling of an h3... so you want it to be subheading?

Reply:
It doesn't have h3 styling on minor nor does it have h2 styling now that you've updated it. You actually need to add the classes, the tags aren't enough.

Reply:
The class doesn't get added for some reason

Reply:
I just noticed the class was not getting added in basic html mode. It's done now.